ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1835,25 +1835,23 @@ tests:
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       E2E_RUN_TAGS: '@amd64 and @azure-ipi and @network-ovnkubernetes and not @fips'
-      OS_IMAGE_MASTERS: azureopenshift:aro4:413_v2:413.92.20230614
-      OS_IMAGE_MASTERS_PLAN: NoPurchasePlan
+      OS_IMAGE_VERSION: 413.92.2023101700
       TAG_VERSION: '@4.14'
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-marketplace
-- as: azure-ipi-marketplace-f28-destructive
-  cron: 32 7 27 * *
+- as: azure-ipi-marketplace-noplan-f28-destructive
+  cron: 24 12 16 * *
   steps:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       E2E_RUN_TAGS: '@amd64 and @azure-ipi and @network-ovnkubernetes and not @fips'
-      OS_IMAGE_MASTERS: azureopenshift:aro4:413_v2:413.92.20230614
-      OS_IMAGE_MASTERS_PLAN: NoPurchasePlan
+      OS_IMAGE_VERSION: 413.92.2023101700
```

Comment:
We should cover `azureopenshift:aro4:413_v2:413.92.20230614` on this job, right?